drift/openapi/admin_api.spec.yaml

```diff
@@ -0,0 +1,36 @@
+---
+openapi: 3.0.1
+info:
+  version: "0.1"
+  title: Drift Backend Service Admin Plane
+  description: Admin API for Drift Service
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: "{{ path_prefix }}{{ app_name }}/admin"
+
+
+paths:
+  /status:
+    get:
+      summary: admin data call
+      operationId: drift.views.admin.status
```

Comment:
if so on my above comment about version, don't forget to add the version in here as well as `drift.admin_views.v1.status`

And I do think you need your own separate views like internal_views and mgmt_views, you would now have admin_views folder and v1.py within that.